app/views/vendor_outage/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render(
+      'idv/shared/error',
+      title: t('vendor_outage.working'),
+      heading: t('vendor_outage.working'),
```

Comment:
Minor: `_error.html.erb` will default `title` to `heading` if omitted:

```suggestion
      heading: t('vendor_outage.working'),
```

https://github.com/18F/identity-idp/blob/db26ec200ab782e7598dd114cb8307d3ceaed988/app/views/idv/shared/_error.html.erb#L5